internal/elasticsearch/securitycontextconstraints.go

```diff
@@ -0,0 +1,40 @@
+package elasticsearch
+
+import (
+	"context"
+
+	securityv1 "github.com/openshift/api/security/v1"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/ViaQ/logerr/kverrors"
+	"github.com/openshift/elasticsearch-operator/internal/manifests/securitycontextconstraints"
+)
+
+// CreateOrUpdateServiceAccount ensures the existence of the serviceaccount for Elasticsearch cluster
+func (er *ElasticsearchRequest) CreateOrUpdateSecurityContextConstraints() error {
+	dpl := er.cluster
+
+	builder := securitycontextconstraints.New("elasticsearch-scc", false, true, false)
+	builder.WithForbiddenSysctls([]string{"*"})
+	builder.WithAllowPrivilegeEscalation(false)
+	builder.WithDefaultAllowPrivilegeEscalation(false)
+	builder.WithVolumes([]securityv1.FSType{"configMap", "secret", "emptyDir", "persistentVolumeClaim"})
+	builder.WithRequiredDropCapabilities([]corev1.Capability{"CHOWN", "DAC_OVERRIDE", "FSETID", "FOWNER", "SETGID", "SETUID", "SETPCAP", "NET_BIND_SERVICE", "KILL"})
+	builder.WithRunAsUserOptions(securityv1.RunAsUserStrategyOptions{
+		Type: securityv1.RunAsUserStrategyRunAsAny,
+	})
+	builder.WithSELinuxContextOptions(securityv1.SELinuxContextStrategyOptions{
+		Type: securityv1.SELinuxStrategyRunAsAny,
+	})
```

Comment:
Security settings are very hard to read and maintain over time. I suggest you add per line a two-liner comment to explain why we set this as is and what's the impact on the ES container.